docs/3.0rc/develop/write-flows.mdx

```diff
@@ -3,41 +3,42 @@ title: Write and run flows
 description: Learn the basics of defining and running flows.
 ---
 
-Flows are the most central Prefect objects. 
-A flow is a container for workflow logic as code. 
+**A flow is a container for workflow logic.**
 
 Flows are defined as Python functions. 
-They can take inputs, perform work, and return an output.
+They can take inputs, perform work, and return a result.
 
-Make any function a Prefect flow by adding the `@flow` decorator to it:
+Make any Python function a Prefect flow by adding the `@flow` decorator to it:
 
 ```python
 from prefect import flow
 
 
 @flow
-def my_flow():
-    return
```

Comment:
i think all examples (within reason) should be directly copy / paste-able and runnable